lib/saml_idp/configurator.rb

```diff
@@ -1,35 +1,18 @@
-# encoding: utf-8
 require 'ostruct'
 module SamlIdp
   class Configurator
-    attr_accessor :x509_certificate
-    attr_accessor :secret_key
-    attr_accessor :password
-    attr_accessor :algorithm
-    attr_accessor :organization_name
-    attr_accessor :organization_url
-    attr_accessor :base_saml_location
-    attr_accessor :entity_id
-    attr_accessor :reference_id_generator
-    attr_accessor :attribute_service_location
-    attr_accessor :single_service_post_location
-    attr_accessor :single_logout_service_post_location
-    attr_accessor :remote_logout_service_post_location
-    attr_accessor :attributes
-    attr_accessor :service_provider
-    attr_accessor :pkcs11
-    attr_accessor :cloudhsm_enabled
-    attr_accessor :cloudhsm_pin
+    attr_accessor :x509_certificate, :secret_key, :password, :algorithm, :organization_name,
+                  :organization_url, :base_saml_location, :entity_id, :reference_id_generator, :attribute_service_location, :single_service_post_location, :single_logout_service_post_location, :remote_logout_service_post_location, :attributes, :service_provider, :pkcs11, :cloudhsm_enabled, :cloudhsm_pin
```

Comment:
did we add a line length cap? this definitely goes over it. I would consider 100 chars to match IDP

Reply:
ah yeah i noted this too -- i didn't add any/change and the defined rules (except to update the names of the ones that had changed/deprecated) so i can add this.

Reply:
hm, there is a line list cap. i wonder if that is considered an unsafe fix, so it collected all the attributes but didn't do the line cap.

Reply:
fixed in https://github.com/18F/saml_idp/pull/98/commits/2b8c1afa357e64a05ddf8932dfae209da2a40a6a and i did confirm that this doesn't fix itself with the `-a` option 🫠